src/content/learn/queueing-a-series-of-state-updates.md

```diff
@@ -1,23 +1,23 @@
 ---
-title: Queueing a Series of State Updates
+title: Mengantre Serangkaian Pembaruan State
 ---
 
 <Intro>
 
-Setting a state variable will queue another render. But sometimes you might want to perform multiple operations on the value before queueing the next render. To do this, it helps to understand how React batches state updates.
+Mengatur variabel *state* akan menambahkan antrean *render* baru. Terkadang Anda ingin melakukan beberapa operasi terhadap nilai sebelum menambahkan antrean *render* selanjutnya. Untuk melakukanya, penting untuk memahami bagaimana React melakukan pembaruan *state* secara berkelompok. 
```

Comment:
Introduce a new word along with the original English word. So we have a better context.

```suggestion
Mengatur variabel *state* akan menambahkan antrean (*queue*) *render* baru. Terkadang Anda ingin melakukan beberapa operasi terhadap nilai sebelum menambahkan antrean *render* selanjutnya. Untuk melakukanya, penting untuk memahami bagaimana React melakukan pembaruan *state* secara berkelompok. 
```

Reply:
Sure, thanks for the early review

Reply:
@milhamm Sure thing, you're welcome